audiomentations/augmentations/transforms.py

```diff
@@ -1094,17 +1121,32 @@ def apply(self, samples, sample_rate):
                 end_sample_index = num_samples
 
             clean_rms = calculate_rms(samples[start_sample_index:end_sample_index])
-            noise_rms = calculate_rms(noise_samples)
+            noise_rms = calculate_rms_without_silence(noise_samples, sample_rate)
```

Comment:
This changes the default behavior of AddShortNoises.

Maybe we can have a parameter like `include_silence_in_noise_rms_estimation` (boolean) for choosing between `calculate_rms` and `calculate_rms_without_silence`?

The idea is that it would be nice to keep AddShortNoises backwards-compatible for existing users of audiomentations, so at this time include_silence_in_noise_rms_estimation should ideally default to True. Add a docstring that explains what the parameter does and when it is useful (when the noises are non-stationary and have silent parts in them). Then users who know what they are doing can set the value to False